contracts/core/types/Types.sol

```diff
@@ -47,5 +47,5 @@ struct SourceStamp {
 
 struct RecipientData {
```

Comment:
if this is used in an event we cant change without a breaking change

Reply:
It does not affect any event, all good. Also types stays the same